### PR TITLE
pangolin - use seqera containers + nf-test; rm pytests

### PR DESCRIPTION
**Pull request by sateeshperi**
opted for seqera conatiners as there was a md5 mismatch on conda with biocontainers

### CODE REVIEW

**Comment by sateeshperi** [comment]
getting this error with seqera container

```
    ****
    Pangolin running in usher mode.
    ****
    Maximum ambiguity allowed is 0.3.
    ****
    Query file: genome.fasta
    ****
    Data files found:
    usher_pb:   /opt/conda/lib/python3.8/site-packages/pangolin_data/data/lineageTree.pb
    ****
  
  Command error:
    Job counts:
        count   jobs
        1       align_to_reference
        1       all
        1       cache_sequence_assign
        1       create_seq_hash
        1       get_constellations
        1       merged_info
        1       scorpio
        1       sequence_qc
        8
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.8/runpy.py", line 194, in _run_module_as_main
        return _run_code(code, main_globals, None,
      File "/opt/conda/lib/python3.8/runpy.py", line 87, in _run_code
        exec(code, run_globals)
      File "/opt/conda/lib/python3.8/site-packages/snakemake/__main__.py", line 4, in <module>
        main()
      File "/opt/conda/lib/python3.8/site-packages/snakemake/__init__.py", line 2007, in main
        config = parse_config(args)
      File "/opt/conda/lib/python3.8/site-packages/snakemake/__init__.py", line 791, in parse_config
        assert v is not None
    AssertionError
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.8/runpy.py", line 194, in _run_module_as_main
        return _run_code(code, main_globals, None,
      File "/opt/conda/lib/python3.8/runpy.py", line 87, in _run_code
        exec(code, run_globals)
      File "/opt/conda/lib/python3.8/site-packages/snakemake/__main__.py", line 4, in <module>
        main()
      File "/opt/conda/lib/python3.8/site-packages/snakemake/__init__.py", line 2007, in main
        config = parse_config(args)
      File "/opt/conda/lib/python3.8/site-packages/snakemake/__init__.py", line 791, in parse_config
        assert v is not None
    AssertionError
    Exiting because a job execution failed. Look above for error message
```

File: modules/nf-core/pangolin/main.nf [hunk 1]
```diff
@@ -4,8 +4,8 @@ process PANGOLIN {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/pangolin:4.2--pyhdfd78af_1' :
-        'biocontainers/pangolin:4.2--pyhdfd78af_1' }"
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/71/7133e48e32b7baa883b8805e600a143b65316e8a014458b8eb4429eb3596f165/data' :
+        'community.wave.seqera.io/library/pangolin:4.2--3bc97f62c0074e70' }"
```
Comment (Joon-Klaps) [inline modules/nf-core/pangolin/main.nf:8]:
```suggestion
        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/9b/9bde3b7cd87b98001f078099e036d442ca034e9bca0d2a1d900dfa628577eab9/data' :
        'community.wave.seqera.io/library/pangolin_snakemake:5282f2aa7d83db6b }"
```

**Review by Joon-Klaps — COMMENTED** [review]
Don't forget to patch the conda `environment.yml` file as well. (to version 4.3)

File: modules/nf-core/pangolin/tests/main.nf.test.snap [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            {
+                "0": [
+                    
+                ],
+                "1": [
+                    
+                ],
+                "report": [
+                    
+                ],
```
Comment (sateeshperi) [inline modules/nf-core/pangolin/tests/main.nf.test.snap:36]: need to regenerate the snapshot @Joon-Klaps

**Review by vagkaratzas — APPROVED** [review]
LGTM! I am not sure about the way to write the seqera containers yet, but if it passes the tests, must be fine ;)

**Comment by sateeshperi** [comment]
@edmundmiller u assigning me a merged PR ?

**Pull request merged**